### DISCUSSION TITLE
email notifications for users/admins are not working

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3448] [Q&A]
Hello!

I have configured a pretix system in my company's premises, but it seems like the e-mail notifications functionality is not working for the users accounts for any notification type that is configured in User Settings->Notifications.
The customer receives an email without any problem, but the user/admin of pretix are not
I am getting the error message below 
{
  "invoices": [],
  "message": "Max retries exceeded",
  "recipient": "",
  "subject": "Internal error"
}

Thank you in advance

**Comment by raphaelm** [comment]
Can you check if pretix.log or the output of pretix-worker contains more details on the error?

**Comment by alemao8** [comment]
There is no error on pretix.log 

In pretix-worker there is something that seems relevant

Jul 10 16:42:08 mkt01-app-02 celery[2070657]: [2023-07-10 13:42:08,298: INFO/ForkPoolWorker-1] Task pretix.base.services.cart.add_items_to_cart[3b74370a-5bfd-4cbb-9c21-b1b5966314af] succeeded in 0.19964560121297836s: None
Jul 10 16:43:34 mkt01-app-02 celery[2070657]: [2023-07-10 13:43:34,521: ERROR/ForkPoolWorker-1] Task pretix.base.services.mail.mail_send_task[ce8b3084-cc5a-4851-a77c-f28e42899420] raised unexpected: SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)')
Jul 10 16:43:34 mkt01-app-02 celery[2070657]: Traceback (most recent call last):
Jul 10 16:43:34 mkt01-app-02 celery[2070657]:   File "/var/pretix/venv/lib64/python3.9/site-packages/pretix/base/services/mail.py", line 602, in mail_send_task
Jul 10 16:43:34 mkt01-app-02 celery[2070657]:     self.retry(max_retries=5, countdown=[10, 30, 60, 300, 900, 900][self.request.retries])
Jul 10 16:43:34 mkt01-app-02 celery[2070657]:   File "/var/pretix/venv/lib64/python3.9/site-packages/celery/app/task.py", line 737, in retry
Jul 10 16:43:34 mkt01-app-02 celery[2070657]:     raise self.MaxRetriesExceededError(
Jul 10 16:43:34 mkt01-app-02 celery[2070657]: celery.exceptions.MaxRetriesExceededError: Can't retry pretix.base.services.mail.mail_send_task

**Reply by raphaelm** [reply]
Yes, this looks like the SSL connection between pretix and your mail server doesn't work. Maybe you have configured pretix to use SSL on a port that actually uses STARTTLS?

**Comment by alemao8** [comment]
I managed to work with this error and now I am not taking any more errors. The pretix-worker logs also are not displaying any errors.
.......................
Sep 07 17:56:48 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:48,404: INFO/MainProcess] Task pretix.base.services.mail.mail_send_task[2406aeaa-0119-4394-b121-022c62a2ccd4] received
Sep 07 17:56:48 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:48,406: INFO/MainProcess] Task pretix.base.services.mail.mail_send_task[c5dcf85e-7bd9-4660-ae10-c0c086d8d239] received
Sep 07 17:56:48 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:48,409: INFO/MainProcess] Task pretix.base.services.mail.mail_send_task[123fe252-8818-47f4-aec6-b752684fc64f] received
Sep 07 17:56:51 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:51,498: INFO/MainProcess] Task pretix.base.services.cart.add_items_to_cart[08fee4d9-bea4-4254-869b-a8275fbbd366] received
Sep 07 17:56:51 mkt01-app-02 celery[2228823]: [2023-09-07 14:56:51,770: INFO/ForkPoolWorker-1] Task pretix.base.services.cart.add_items_to_cart[08fee4d9-bea4-4254-869b-a8275fbbd366] succeeded in 0.2703020013868809s: None
Sep 07 17:56:55 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:55,566: INFO/MainProcess] Task pretix.base.services.cart.clear_cart[92c80e24-4e74-4876-8ead-6c56628f1671] received
Sep 07 17:56:55 mkt01-app-02 celery[2228823]: [2023-09-07 14:56:55,732: INFO/ForkPoolWorker-1] Task pretix.base.services.cart.clear_cart[92c80e24-4e74-4876-8ead-6c56628f1671] succeeded in 0.16491420194506645s: None
Sep 07 17:56:59 mkt01-app-02 celery[2228820]: [2023-09-07 14:56:59,305: INFO/MainProcess] Task pretix.base.services.cart.add_items_to_cart[b9e16ff6-a557-4716-a8d1-ef1c710c3bfb] received
Sep 07 17:56:59 mkt01-app-02 celery[2228823]: [2023-09-07 14:56:59,542: INFO/ForkPoolWorker-1] Task pretix.base.services.cart.add_items_to_cart[b9e16ff6-a557-4716-a8d1-ef1c710c3bfb] succeeded in 0.23618339747190475s: None
Sep 07 17:57:16 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:16,634: INFO/MainProcess] Task pretix.base.services.orders.perform_order[a1bc9fdf-d103-4a8a-bd31-8ca4d1293178] received
Sep 07 17:57:16 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:16,775: INFO/MainProcess] Task pretix.base.services.notifications.notify[3647d436-5f04-42f1-8bd2-73cf83d1f42a] received
Sep 07 17:57:16 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:16,778: INFO/MainProcess] Task pretix.api.webhooks.notify_webhooks[e2814d5c-77c2-4207-ab49-92d30e3bc4b8] received
Sep 07 17:57:16 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:16,871: INFO/MainProcess] Task pretix.base.services.notifications.send_notification[90a3c743-2177-4ef1-b9d1-4e4ecad8f0ed] received
Sep 07 17:57:16 mkt01-app-02 celery[2228824]: [2023-09-07 14:57:16,874: INFO/ForkPoolWorker-2] Task pretix.base.services.notifications.notify[3647d436-5f04-42f1-8bd2-73cf83d1f42a] succeeded in 0.09573880210518837s: None
Sep 07 17:57:16 mkt01-app-02 celery[2228824]: [2023-09-07 14:57:16,891: INFO/ForkPoolWorker-2] Task pretix.api.webhooks.notify_webhooks[e2814d5c-77c2-4207-ab49-92d30e3bc4b8] succeeded in 0.012920603156089783s: None
Sep 07 17:57:17 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:17,233: INFO/MainProcess] Task pretix.base.services.mail.mail_send_task[80d54b1e-aa65-4e6f-84be-bc9dc264a18a] received
Sep 07 17:57:17 mkt01-app-02 celery[2228824]: [2023-09-07 14:57:17,235: INFO/ForkPoolWorker-2] Task pretix.base.services.notifications.send_notification[90a3c743-2177-4ef1-b9d1-4e4ecad8f0ed] succeeded in 0.34134020283818245s: None
Sep 07 17:57:17 mkt01-app-02 celery[2228820]: [2023-09-07 14:57:17,453: INFO/MainProcess] Task pretix.base.services.mail.mail_send_task[88cb9aa1-ab01-4a7b-841a-4b50e99689ce] received
Sep 07 17:57:17 mkt01-app-02 celery[2228823]: [2023-09-07 14:57:17,458: INFO/ForkPoolWorker-1] Task pretix.base.services.orders.perform_order[a1bc9fdf-d103-4a8a-bd31-8ca4d1293178] succeeded in 0.821730799973011s: {'order_id': 207, 'warnings': []}
Sep 07 17:57:18 mkt01-app-02 celery[2228823]: [2023-09-07 14:57:18,656: INFO/ForkPoolWorker-1] Task pretix.base.services.mail.mail_send_task[88cb9aa1-ab01-4a7b-841a-4b50e99689ce] succeeded in 1.1956050023436546s: None
.....................

Also the pretix.cfg file has the below settings
[mail]
; See config file documentation for more options
from=<from address>
host=<host name>
port=465
tls=off
ssl=on
user=<username>
password=<password>

I cannot figure out what goes wrong and still I am not receiving any emails for new orders, although the customers do receive e-mails about their purchases.

**Comment by alemao8** [comment]
solved!
port=587
tls=on
